scalafix-core/shared/src/main/scala/scalafix/internal/util/TypeSyntax.scala

```diff
@@ -13,7 +13,7 @@ object TypeSyntax {
       implicit index: SemanticdbIndex): (Type, Patch) = {
```

Comment:
This could be useful:
https://github.com/lihaoyi/PPrint/blob/master/pprint/shared/src/main/scala/pprint/TPrint.scala
https://github.com/lihaoyi/PPrint/blob/master/build.sbt#L53